<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBackupDatabaseSettings</class>
 <widget class="QDialog" name="FormBackupDatabaseSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Backup database/settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="m_groupFile">
     <property name="title">
      <string>Output directory</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="m_btnSelectFolder">
        <property name="text">
         <string>&amp;Select directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LabelWithStatus" name="m_lblSelectFolder" native="true">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_groupFeeds">
     <property name="title">
      <string>Backup properties</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Items to backup</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="m_checkBackupDatabase">
        <property name="text">
         <string>Database</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="m_checkBackupSettings">
        <property name="text">
         <string>Settings</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Backup name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="LineEditWithStatus" name="m_txtBackupName" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Operation results</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="LabelWithStatus" name="m_lblResult" native="true">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditWithStatus</class>
   <extends>QWidget</extends>
   <header>lineeditwithstatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabelWithStatus</class>
   <extends>QWidget</extends>
   <header>labelwithstatus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormBackupDatabaseSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
